src/lambda/models/samLambdaRuntime.ts

```diff
@@ -88,3 +96,58 @@ export function getRuntimeFamily(langId: string): RuntimeFamily {
 export function getDefaultRuntime(runtime: RuntimeFamily): string | undefined {
     return DEFAULT_RUNTIMES.get(runtime)
 }
+
+/**
+ * Returns a set of runtimes for a specified runtime family or undefined if not found.
+ * @param family Runtime family to get runtimes for
+ */
+function getRuntimesForFamily(family: RuntimeFamily): Set<Runtime> | undefined {
+    switch (family) {
+        case RuntimeFamily.NodeJS:
+            return nodeJsRuntimes
+        case RuntimeFamily.Python:
+            return pythonRuntimes
+        case RuntimeFamily.DotNetCore:
+            return dotNetRuntimes
+        default:
+            return undefined
+    }
+}
+
+/**
+ * Creates a quick pick for a Runtime with the following parameters (all optional)
+ * @param params buttons: array of buttons to add to the quick pick;
+ * currRuntime: Runtime to set a "Selected Previously" mark to;
+ * runtimeFamily: a RuntimeFamily that will define the list of runtimes to show (default: samLambdaCreatableRuntimes)
```

Comment:
Does docstring format have a better way to document these "keywords style" params?

Maybe this works as a hack:

```
@param buttons ...
@param currRuntime ...
```

Reply:
Looks like this is the correct way to do it: https://jsdoc.app/tags-param.html#parameters-with-properties

Reply:
FYI, while that's the technically correct way to do it, it doesn't look like Typescript (and thus, VS Code) has implemented it yet: https://github.com/microsoft/TypeScript/issues/11597